### DISCUSSION TITLE
Consider submitting minijinja to template-benchmarks-rs

### DISCUSSION THREAD

**Discussion by infogulch** [mitsuhiko/minijinja#229] [Ideas]
[rosetta-rs/template-benchmarks-rs](https://github.com/rosetta-rs/template-benchmarks-rs) is a straightforward benchmarking suite designed to compare the performance of various rust templating engines. I think this kind of comparison suite is useful to everyone, including users and template library developers, and I'd like to see minijinja be included.

* Were you aware of the existence of `template-benchmarks-rs`?
* Do you like the way `template-benchmarks-rs` is designed?
* How much effort do you think it would take to open a PR to add minijinja to template-benchmarks-rs?

**Comment by mitsuhiko** [comment]
I wasn’t aware of it and it sounds like easy enough to add. I’m not sure if I have the time myself though.
